### ISSUE TITLE
Update 1.1.0 reports false positives for `no-missing-keys`

### ISSUE THREAD

**Issue by jokay** [intlify/eslint-plugin-vue-i18n#268]
**Tell us about your environment**

- **ESLint version:** `8.2.0`
- **eslint-plugin-vue version:** `8.0.3`
- **eslint-plugin-vue-i18n version:** `1.1.0`
- **Node version:** `16.13.0`

**Please show your full configuration:**

```json5
'@intlify/vue-i18n/key-format-style': 'off',
'@intlify/vue-i18n/no-duplicate-keys-in-locale': 'error',
'@intlify/vue-i18n/no-dynamic-keys': 'off',
'@intlify/vue-i18n/no-missing-keys': 'error',
'@intlify/vue-i18n/no-raw-text': 'off',
'@intlify/vue-i18n/no-unused-keys': [
    'off',
    {
    extensions: [
        '.js',
        '.ts',
        '.vue'
    ]
    }
],
```

**What did you do?**

##### en-US.json

```json
{
    "debug": {
        "infobar": {
            "version": "version"
        }
    }
}
```

##### DebugBar.vue

```vue
{{ $t('debug.infobar.version') }}
```

For me it looks like it can't handle multiple levels in the i18n translation files?

**What did you expect to happen?**

Should not report errors, was working on `1.0.1`.

**What actually happened?**

It reports errors.

```sh
5:12  error  'debug.infobar' does not exist in localization message resources  @intlify/vue-i18n/no-missing-keys
```

**Repository to reproduce this issue**

Sorry I can't provide a repo for this, feel free to verify yourself and fix it or close the issue if you don't care.

**Comment by ota-meshi** [comment]
I can't reproduce the issue. If there is no repo that can reproduce the issue, nothing can be done and the issue will be closed.

**Comment by jokay** [comment]
Ok, it was working on **1.0.1** no more one **1.1.0** but I understand, so close the issue :wink:

**Comment by lukashass** [comment]
This happened to me on `1.2.0` with one complete locale and an empty locale file:

`error  'message' does not exist in localization message resources  @intlify/vue-i18n/no-missing-keys`

```jsonc
// en.json
{
  "end": "End",
  "message": {
    "cancel": "Cancel",
    "description": "Do you want to delete this booking ?",
    "label": "Delete"
  }
}
```

```jsonc
// de.json
{}
```

Just removing the empty locale fixed this problem.
Hope this helps pinpointing the issue.
